docs/jeo-objects.md

```diff
@@ -0,0 +1,83 @@
+# Full List of Jeo Objects
```

Comment:
@volodya-lombrozo why can't you put this list into the README? It's not yet too big for that.

Reply:
@volodya-lombrozo also, maybe it's better to generate this list by a script executed by GitHub Actions and inject into the README. This will guarantee it's correctness.